docs/integrations/gitlab.md

```diff
@@ -0,0 +1,74 @@
+---
+title: GitLab Integration
+sidebar_label: GitLab
+---
+
```

Comment:
You should add a link to this page from the table on the Integrations landing page (pipes/docs/integrations) also.  (if you could add a link for the slack integration too it would be appreciated - that is also missing)

Reply:
Done